claim.go

```diff
@@ -91,20 +90,20 @@ func (sc SchemaHash) MarshalText() ([]byte, error) {
 	return dst, nil
 }
 
-// DataSlot length is 32 bytes. But not all 32-byte values are valid.
+// ElementBytes length is 32 bytes. But not all 32-byte values are valid.
 // The value should be not greater than Q constant
 // 21888242871839275222246405745257275088548364400416034343698204186575808495617
-type DataSlot [32]byte
+type ElementBytes [32]byte
```

Comment:
Would be good to move ElementBytes and functions related to it to a separate file (ElementBytes.ToInt, NewElementBytesFromInt, etc)